src/version.ts

```diff
@@ -88,19 +74,13 @@ async function fetchLatestVersion(
 export async function getLatestRemoteVersion(
   config: Config = getConfig()
 ): Promise<string> {
-  const response: GraphQlQueryResponseData = await fetchLatestVersion(config);
-
-  if (response.repository.packages.nodes.length > 1) {
-    throw new Error(
-      `🚧 Found ${response.repository.packages.nodes.length} packages on '${config.repository}', expected only 1.  Monorepo/multiple packages is not supported yet.`
-    );
-  }
+  const { data } = await fetchLatestVersion(config);
```

Comment:
👍🏻 Yip, matches [examples](https://github.com/octokit/core.js#authentication), we get a `data` object back.